src/slack/slack.js

```diff
@@ -871,12 +876,13 @@ export class Slack {
     const totalCount = Array.from(order.items.entries()).reduce((acc, entry) => acc + entry[1], 0)
 
     if (totalCount === 0) {
-      return {...order, orderConfirmation: null}
+      return null
```

Comment:
anyway... why not put the `say` right here? instead of having a clear connection that
`totalCount is 0` ➡️  `show error message about links`
we have
`totalCount is 0` ➡️  `null is returned` ➡️  `null is saved as user's order` ➡️  `if order is null, show error message about links`.
maybe I don't see a reason, but I find it unnecessarily confusing.
if you agree to move the `say` here, we can then also create a local constant for the error message and reuse it below

Reply:
Initially, I also thought about this solution but did not want to pass `say` through functions :

> handleUserMessage(say) -> updateOrder(say)

If it is ok, I can pass and call `say` in `updateOrder()`

Reply:
yeah, thought this could be the reason. it's OK, let's pass it